packages/quantic/force-app/main/default/lwc/quanticResultText/quanticResultText.js

```diff
@@ -79,7 +79,9 @@ export default class QuanticResultText extends LightningElement {
    */
   get fieldValue() {
     // @ts-ignore
-    return this.field ? this.result?.raw[this.field] : undefined;
+    return this.field
```

Comment:
you could also do something like this I believe:
`return this.field && (this.result?.raw[this.field] ?? this.result?.[this.field]);`

Reply:
could be yeah, but we don't really want the `fieldValue` to return a boolean value false. we want it to either return the value or nothing aka `undefined`.